quill-cassandra-ce/src/test/scala-2.12/io/getquill/context/cassandra/catEffect/package.scala

```diff
@@ -0,0 +1,16 @@
+package io.getquill.context.cassandra
+
+import io.getquill.util.LoadConfig
+import io.getquill.{ CassandraCeContext, CassandraContextConfig, Literal }
+import cats.effect.{ Async, IO }
+import io.getquill.context.cassandra.encoding.{ Decoders, Encoders }
+
+package object catsEffect {
+
+  lazy val testCeDB: CassandraCeContext[Literal.type, IO] = {
+    implicit val af = Async[IO]
```

Comment:
This seems kind of unnecessary. You don't have to capture the `Async[IO]`, you can just ask for it any time you want it. Unless there's some type witchcraft here that I'm not aware of?

Reply:
Don't you need it for the `new CassandraCeContext` below? I imagine you could also pass it explicitly?
```scala
    new CassandraCeContext(Literal, c.cluster, c.keyspace, c.preparedStatementCacheSize)(Async[IO]) with CassandraTestEntities with Encoders with Decoders
```

Reply:
I went with inlining Async[F] to avoid following error:
```
[error] /app/quill-cassandra-ce/src/test/scala-2.12/io/getquill/context/cassandra/catEffect/package.scala:14:9: diverging implicit expansion for type cats.effect.Async[F]
[error] starting with method asyncForKleisli in object Async
[error]     new CassandraCeContext(Literal, c.cluster, c.keyspace, c.preparedStatementCacheSize) with CassandraTestEntities with Encoders with Decoders
[error]         ^
[error] /app/quill-cassandra-ce/src/test/scala-2.12/io/getquill/context/cassandra/catEffect/package.scala:14:5: type mismatch;
[error]  found   : <error> with io.getquill.context.cassandra.CassandraTestEntities with io.getquill.context.cassandra.encoding.Encoders with io.getquill.context.cassandra.encoding.Decoders
[error]  required: io.getquill.CassandraCeContext[io.getquill.Literal.type,cats.effect.IO]
[error]     new CassandraCeContext(Literal, c.cluster, c.keyspace, c.preparedStatementCacheSize) with CassandraTestEntities with Encoders with Decoders
```
When using cats.effect.syntax._